style: add telegram style bubble

Sinofine opened the pull request.

Sinofine committed:
style: add telegram style bubble

lilydjwg commented:
头像 sticky 一下？不知道那个角能不能也 sticky？

Sinofine commented:
> 头像 sticky 一下？不知道那个角能不能也 sticky？

目前那个角是absolute排版，不过可以去掉（是为了之前靠底准备的）
应该都可以做，等我下课改一下
不过如果要上下浮动的话可能就不能做有弧度的角了（需要更多element，目前是两个pseudo element做的一侧弧度一侧平）
另外telegram本身角不是sticky样式，头像是sticky

lilydjwg commented:
我也不知道角跟着 sticky 好不好看，容易的话可以弄出来试试，难的话就算了。另外这个灰色背景在亮色上有点太沉了，可能调亮一点比较好看？

另外对于超长昵称，你有办法让昵称不折行、伸出去，但是消息框和内容不要跟着伸到外面去吗？比如[这里靠后](https://luoxu-web-pnbjkesef-lilydjwg.vercel.app/#g=1094615131&q=%E5%A5%BD%E8%80%B6)。

Sinofine commented:
按原来的设计角是不方便sticky的，头像可以sticky
![图片](https://user-images.githubusercontent.com/47494321/136500397-deab6b1e-4bff-4cb3-8e79-60b07056a118.png)
是这种效果吗？如果是的话得用border重新设计一下角的显示了(也不sticky)
这是利用了overflow的截断和position:absolute不占位的特性
另外灰色是`#eee`,下面是`#f9f9f9`
![图片](https://user-images.githubusercontent.com/47494321/136500579-ca153f44-0c0d-491c-a7fa-e7f49e7b142a.png)

之所以这么别扭主要是因为不想为了排版而引入新的DOM层级，否则可以做的更好点

Sinofine commented:
![图片](https://user-images.githubusercontent.com/47494321/136502718-ee160ead-44d8-433c-aea7-5760410c0cee.png)
这个效果如何？
因为太短了也不好看，所以我设置message宽度至少有7em宽。

Sinofine committed:
style: refactor

change message background to #f9f9f9; make username overflown; make
avatar sticky;

Sinofine committed:
style: slightly decrease margin between username and message

Sinofine committed:
style: slightly increase margin& change background to #f2f2f2

lilydjwg commented:
如果没有到达页面边界，长昵称不要截短了。其实我更希望昵称完全不截短，只是放它往右边伸出去。因为我在 Telegram 之类的地方老是遇到想看的东西跑到 … 里看不到很心烦。

Sinofine commented:
> 如果没有到达页面边界，长昵称不要截短了。其实我更希望昵称完全不截短，只是放它往右边伸出去。因为我在 Telegram 之类的地方老是遇到想看的东西跑到 … 里看不到很心烦。

这个简单...就是不好看😂，伸出边界

lilydjwg commented:
我那个版本没弄出来，会把容器也撑大，导致消息也不在窗口边缘换行了。

Sinofine commented:
> 我那个版本没弄出来，会把容器也撑大，导致消息也不在窗口边缘换行了。

防止撑大可以利用position absolute

lilydjwg commented:
呃，我都忘记这个 pr 了。是放弃进一步的修改了么？

Copay commented:
> 呃，我都忘记这个 pr 了。是放弃进一步的修改了么？

好久没做这方面的工作了，似乎项目后续做了一些更改，以后再说吧。
~~啊上错号了~~
